internal/pkg/install/prepare.go

```diff
@@ -62,6 +62,11 @@ func Prepare(data *userdata.UserData) (err error) {
 		return nil
 	}
 
+	if data.Install.IsInstalled() {
```

Comment:
Should we do `data.Install.IsInstalled() && !data.Install.Force { skip }`?

Reply:
Force is a bit different and gets checked later.